lib/build_cache.py

```diff
@@ -691,6 +721,11 @@ def checkout(self, image, git_hash, base_image):
       self.worktree_add(image, git_hash)
       self.git_restore(image.unpack_path, [], False)
 
+   def checkout_ready(self, image, git_hash, base_image=None):
+      # Wrapper for “checkout()” followed by “ready()”
```

Comment:
This comment is redundant with the code. Either explain why, or remove the comment.